packages/apps/dev-wallet/src/Components/PluginIcon/PluginIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import {
+  pluginIconClass,
+  smallPluginIconClass,
+} from '@/pages/plugins/style.css';
+import { getInitials } from '@/utils/get-initials';
+
+export const PluginIcon = ({
+  name,
+  size = 'md',
+}: {
+  name: string;
+  size?: 'md' | 'sm';
+}) => (
+  <div
+    x-testId={name}
```

Comment:
what is this used for? I dont see it anywhere used.
can we call it `data-testid`. so we can use it for e2e?